bigflow/build/dist.py

```diff
@@ -267,6 +307,11 @@ def project_setup(
     setup(project_setup(**auto_configuration('my_super_project')))
     '''
 
+    _maybe_dump_setup_params({
+        'name': project_name,
+    })
+    bigflow.build.pip.maybe_recompile_requirements_file(project_requirements_file)
```

Comment:
After recompilation print a gigantic (multi line) message to inform that one needs to reinstall packages (and add the instruction to do it, `pip install ...`).

Reply:
Done.